core/sys/linux/wrappers.odin

```diff
@@ -54,22 +54,45 @@ WCOREDUMP :: #force_inline proc "contextless" (s: u32) -> bool {
 // TODO: sigaddset etc
 
 
-/// Iterate the results of getdents
-/// Only iterates as much data as loaded in the buffer
-/// In case you need to iterate *all* files in a directory
-/// consider using dirent_get_iterate
-///
-/// Example of using dirent_iterate_buf
-///   // Get dirents into a buffer
-///   buf: [128]u8
-///   sys.getdents(dirfd, buf[:])
-///   // Print the names of the files
-///   for dir in sys.dirent_iterate_buf(buf[:], &offs) {
-///       name := sys.dirent_name(dir)
-///       fmt.println(name)
-///   }
-/// This function doesn't automatically make a request
-/// for the buffer to be refilled
+/*
+Iterate the results of `getdents()`.
+
+This procedure extracts a directory entry from `buf` at the offset `offs`.
+`offs` will be modified to store an offset to the possible next directory entry
+in `buf`. The procedure only iterates as much data as loaded in the buffer and
+does not automatically make a request for the buffer to be refilled.
+
+**Inputs**:
+- `buf` - byte buffer with data from `getdents()`
+- `offs` - offset to the next possible directory entry in `buf`
+
+**Returns**:
+- A pointer to a directory entry in `buf`, or `nil`.
+- A bool value denoting if a valid directory entry is returned.
+
+**Example**:
```

Comment:
This should be `Example:` or the documentation generator needs to be changed to pick up on `**Example:**` too.

Did we actually decide on the `**`? I still think it's a bad idea.

Reply:
Welp, the toplevel post in https://github.com/odin-lang/Odin/discussions/5039 uses `**Example**:`, but grepping the sources shows me that `Example:` is used most of the time. Will update.